client/my-sites/people/team-list/index.jsx

```diff
@@ -1,158 +1,38 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:my-sites:people:team-list' ),
-	omit = require( 'lodash/omit' );
+import React, { Component } from 'react';
 
 /**
  * Internal dependencies
  */
-var Card = require( 'components/card' ),
-	PeopleListItem = require( 'my-sites/people/people-list-item' ),
-	SiteUsersFetcher = require( 'components/site-users-fetcher' ),
-	UsersActions = require( 'lib/users/actions' ),
-	InfiniteList = require( 'components/infinite-list' ),
-	deterministicStringify = require( 'lib/deterministic-stringify' ),
-	NoResults = require( 'my-sites/no-results' ),
-	analytics = require( 'lib/analytics' ),
-	PeopleListSectionHeader = require( 'my-sites/people/people-list-section-header' );
+import Team from './team';
+import SiteUsersFetcher from 'components/site-users-fetcher';
 
 /**
  * Module Variables
  */
-var Team = React.createClass( {
-	displayName: 'Team',
+class TeamList extends Component {
+	render() {
+		const { props } = this;
+		const { site, search } = props;
```

Comment:
see note above - it's safe to call `this.props`

```js
const { search, site } = this.props;
```

please also keep catalogues like this in alphabetical order